src/cli/command-registrar.ts

```diff
@@ -235,6 +238,18 @@ export default class CommandRegistrar {
           `warning: '${chalk.bold(subcommand)}' is not a valid command.\nsee 'bit --help' for additional information.\n`
         )
       );
+      const suggestion = didYouMean(subcommand, commander.commands.map(cmd => cmd._name).filter(c => !c.private));
```

Comment:
This filter won't work.
1. you filtering after the map, you should filter it before (after the map you won't have the props but only the name)
2. the private prop is translated to something else during the constructor.
it should be:
`commander.commands.filter(c => !c._noHelp).map(cmd => cmd._name);`